src/pages/Main/SortingSearchArea.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+function SortingSearchArea(props) {
+  return (
+    <SortingSearchWrapper>
+      <MovieSortingWrapper onClick={props.onMovieSortingClick}>
+        {props.sorters.map((sorter, index) => (
+          <MovieSorter>
+            <SortingButton
+              isActive={props.sortActive === index ? true : false}
```

Comment:
```suggestion
              isActive={sortActive === index}
```
sortActive === index 비교 연산 자체가 불리언입니다

Reply:
옙. 쓸데없는 부분 좀 줄여보겠습니다.